PatTools/python/patFinalStateProducers.py

```diff
@@ -62,7 +62,7 @@ def produce_final_states(process, daughter_collections, output_commands,
         if 'extraWeights' in src:
             process.patFinalStateEventProducer.extraWeights = src['extraWeights']
         process.patFinalStateEventProducer.trgSrc = cms.InputTag("selectedPatTrigger")
-        process.patFinalStateEventProducer.rhoSrc = cms.InputTag('fixedGridRhoAll')
+        process.patFinalStateEventProducer.rhoSrc = cms.InputTag('fixedGridRhoFastjetAll')
```

Comment:
It was part of the HTT sync, I will remove it because I don't have documentation.